docs/chapters/ABOUT_ONEKO.md

```diff
@@ -0,0 +1,79 @@
+# About O-Neko
+
+O-Neko is a Kubernetes-native application that allows to cross a bridge between developers and other stakeholders by
+deploying development versions of your software into Kubernetes to allow everybody to try and test them.
+
+## Features
+
+The most important features of O-Neko are:
+
+* Deploy development versions (e.g. every branch) to Kubernetes with one click
+* Configure projects with native Kubernetes .yaml files
+* Automatically re-deploy running versions when the corresponding Docker image has changed (configurable per project and version)
+* Automatically stop running versions after a specific time (configurable per project and version)
+* Project meshes that allow you to wrap multiple projects into one pool and deploy specific versions of them together,
+effectively providing a way to deploy dynamic test server setups with ease
+* Select variables that allow fast changes to frequently used configuration settings (e.g. server URLs)
+
+## Pre-requisites
+
+Generally, nearly every project able to run in Kubernetes can be deployed with O-Neko. To run a project in O-Neko:
+
+* You need Docker containers of all project versions you want to deploy with O-Neko
+* You need to provide the Kubernetes .yaml files in a simple template format to configure a project
+* O-Neko works with kubernetes versions 1.9.0 - 1.15.3
+
+## How does it work?
+
+* O-Neko is running inside your Kubernetes cluster
```

Comment:
That is at least our recommendation. Technically you could have it running outside the cluster. It just needs to be able to access it.